.circleci/config.yml

```diff
@@ -2,10 +2,14 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:8-browsers
     working_directory: ~/repo
     steps:
       - checkout
       - run: yarn install
-      - run: yarn examples &
+      - run:
+          name: start SPAs
+          command: yarn examples
+          background: true
```

Comment:
[![LGTM](http://lgtm.in/p/aZw92cLlq)](http://lgtm.in/i/aZw92cLlq)

[:+1:](http://lgtm.in/u/aZw92cLlq) [:-1:](http://lgtm.in/r/aZw92cLlq)